mysql/resource_grant_test.go

```diff
@@ -888,21 +889,6 @@ func TestAccGrantOnProcedure(t *testing.T) {
 					prepareProcedure(dbName, procedureName),
 				),
 			},
-			{
```

Comment:
What's wrong with this test? I believe it should stay and possibly be fixed if there's something wrong.

I tried running with it and it shows some issues you'd see, if you ran it manually with config.

Reply:
My understanding—but keep in mind I'm not an expert on the matter—is that semantically it makes no sense to provide procedure grants with tables, so the test was not meaningful. I couldn't find any references online to such a scenario either, so please do correct me if I'm missing something. Again, I'm not an expert on this so there's a good chance I'm wrong :)

Reply:
Got it. I couldn't find a way to identify how the object is defined in terraform in the code so that I can build the resource accordingly. Are there any downsides to normalising it? I mean, if the object is parsed correctly and the representation is not crucial, can we just pick one and go with it?

From what I tested, regardless of how the terraform object is defined, we can use either of the formats and it works the same way